### PR TITLE
STORM-675. Allow users to have storm-env.sh under config dir to set custom JAVA_HOME and other env variables.

### PR DESCRIPTION
**Pull request by harshach**

### CODE REVIEW

**Comment by HeartSaVioR** [comment]
AFAIK, STORM-487 was intended to lay the foundation of unifying storm launch script.
Do we have plan to change storm.cmd to use storm.py?

**Comment by harshach** [comment]
@HeartSaVioR yes but wanted to keep the .cmd files for at least a release or two as there might be users who are depend on these. Since storm.py is now windows compatabile we can phase out the current storm.cmd

**Comment by HeartSaVioR** [comment]
Great! Looks good to me.
Btw, it's better to point out storm.py's actual changed lines since it's hard to find difference.

**Comment by harshach** [comment]
@HeartSaVioR bin/storm which used to be a python script now moved to bin/storm.py and in place of bin/storm we added storm shell script to give users a way to configure storm env variables. Because of this renaming files you see the diff like that.

**Comment by HeartSaVioR** [comment]
@harshach I know, I just want to point out that bin/storm is not only renamed but also modified a bit.

**Comment by Parth-Brahmbhatt** [comment]
+1.

**Comment by harshach** [comment]
@ptgoetz can you please take a look at this.

File: bin/storm [hunk 1]
```diff
@@ -1,5 +1,7 @@
-#!/usr/bin/python
-
+#!/bin/bash
+#
+# Copyright 2014 The Apache Software Foundation
```
Comment (ptgoetz) [inline bin/storm]: We shouldn't need the copyright year in the license header.

File: conf/storm-env.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright 2014 The Apache Software Foundation
```
Comment (ptgoetz) [inline conf/storm-env.sh]: Same as previous comment.

**Comment by harshach** [comment]
@ptgoetz Thanks for the review. Removed the year from LICENSE.

**Comment by ptgoetz** [comment]
+1. The diffs of `storm` vs. `storm.py` are minimal and related to windows compatibility.

For easy reference, here they are:

```
$ diff -w storm storm.py
37a38,40
> def is_windows():
>     return sys.platform.startswith('win')
>
174c177
<         JAVA_CMD, jvmtype, get_config_opts(),
---
>         "java", jvmtype, get_config_opts(),
184c187
<     else:
---
>     elif is_windows():
186a190,191
>     else:
>         os.execvp(JAVA_CMD, all_args)
```

**Pull request merged**